week1/index_products.py

```diff
@@ -80,38 +81,100 @@
 
         ]
 
+def create_index(index_name):
+    client = get_opensearch()
+    try:
+        print(client.cat.count(index_name, params={"v": "true"}))
```

Comment:
Might want to use `logger` here instead of printing out since we have that available, like 
logger.info('etc......')
https://realpython.com/python-logging/